src/main/kotlin/ar/edu/unahur/obj2/semillasAlViento/Parcela.kt

```diff
@@ -2,44 +2,41 @@ package ar.edu.unahur.obj2.semillasAlViento
 
 class Parcela(val ancho: Int, val largo: Int, val horasSolPorDia: Int) {
   val plantas = mutableListOf<Planta>()
-  var cantidadPlantas = 0
 
+
+  fun cantidadPlantas()= plantas.size
+  fun parcelaTieneComplicaciones(parcela: Parcela) =
+    parcela.plantas.any { it.horasDeSolQueTolera() < parcela.horasSolPorDia }
```

Comment:
Bien que movieron el método acá, pero:
- Está mal que reciba una parcela: estamos dentro de la clase Parcela, así que `this` ya es una parcela. Fijense cómo lo están usando, `parcela.parcelaTieneComplicaciones(parcela)` en vez de `parcela.parcelaTieneComplicaciones()`
- Siendo que está dentro de `Parcela`, el nombre tendría que ser simplemente `tieneComplicaciones()`